src/main/scala/io/iohk/ethereum/vm/OpCode.scala

```diff
@@ -177,7 +177,7 @@ object OpCode {
   * @param delta number of words to be popped from stack
   * @param alpha number of words to be pushed to stack
   */
-sealed abstract class OpCode(val code: Byte, val delta: Int, val alpha: Int, val constGasFn: FeeSchedule => BigInt) {
+abstract class OpCode(val code: Byte, val delta: Int, val alpha: Int, val constGasFn: FeeSchedule => BigInt) extends Product with Serializable {
```

Comment:
I'd personally go with changing the line that failed to compile rather than adding this requirement here, but it's not a blocker